### DISCUSSION TITLE
Linux old AIRSDK

### DISCUSSION THREAD

**Discussion by ahmed-raboudi** [airsdk/Adobe-Runtime-Support#839] [Q&A]
I have an old application that was running on an old linux machine. The server crashed and we need to reinstall the AdobeAirSDK v2.6 for linux.
Do you have a download link to the linux ADOBEAIRSDK 2.6?

**Comment by ajwfrost** [comment]
Hi - we aren't allowed to provide the old Adobe AIR SDKs (I think because some of the media codecs inside them are no longer covered by the appropriate licenses) - but we are putting together a Linux version of the latest AIR SDK, for use by commercially licensed customers.. feel free to email me at andrew.frost at harman.com and we can check whether this might work for you..

thanks

**Reply by PippoApps** [reply]
This is great news.
Will it be bundled with new releases? How to get hold of the Linux SDK?
thanks
